easybuild/easyconfigs/l/LMfit/LMfit-0.9.14-intel-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'PythonBundle'
+
+name = 'LMfit'
+version = '0.9.14'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://lmfit.github.io/lmfit-py'
+description = "Lmfit provides a high-level interface to non-linear optimization and curve fitting problems for Python"
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+
+dependencies = [('Python', '2.7.15')]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('asteval', '0.9.12', {
+        'source_urls': ['https://pypi.python.org/packages/source/a/asteval'],
+        'checksums': ['38f3b0592cae7e7f65adc687e37aad1824a8e518245603a29ec33258277e779b'],
+    }),
+    ('uncertainties', '3.1.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/u/uncertainties'],
+        'checksums': ['ba07c17a8a78cb58a47cd373079c7ea459f8b26cd474e29163b6ba0d72856a1e'],
+    }),
+    (name, version, {
+        'source_urls': ['https://pypi.python.org/packages/source/l/lmfit'],
+        'source_tmpl': 'lmfit-%(version)s.tar.gz',
+        'checksums': ['5106b9372e5edf809ef6f498dc1588b3cd589aeea7175faf196a867e5a91ae18'],
+    }),
+
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+modextrapaths = {'PYTHONPATH': ['lib/python%(pyshortver)s/site-packages']}
```

Comment:
The addition to `PYTHONPATH` is now automatic with `PythonBundle`. It can be removed.